src/server.ts

```diff
@@ -90,31 +99,56 @@ function throwResponseValidationError(tag: string, originalValue: any, e: oar.Ma
   return null as any;
 }
 
-function voidify(value: {} | undefined) {
-  if (value == null || Object.keys(value).length > 0) {
+function lowercaseObject(o?: {} | undefined | null) {
+  if (o == null) {
```

Comment:
If `o` is undefined, this `if` won't catch it, and then as error will be thrown when `Object.keys(undefined)` gets run

Reply:
``` 
> null == undefined
< true
```

Reply:
oooh, I didn't notice this wasn't a `===` sign. nice!